src/main/java/uk/gov/pay/directdebit/utils/ResponseUtil.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.pay.directdebit.utils;
+
+import com.google.common.collect.ImmutableMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+import static javax.ws.rs.core.Response.Status.BAD_REQUEST;
+import static javax.ws.rs.core.Response.Status.NOT_FOUND;
+import static javax.ws.rs.core.Response.status;
+
+public class ResponseUtil {
```

Comment:
this class should be removed, we are using different patterns in dd connector. BadRequestResponse and NotFoundResponse should use the exception mapper and there's no need to have helpers to serialise an entity

Reply:
👍